simcase-game/public/manifest.json

```diff
@@ -0,0 +1,15 @@
+{
```

Comment:
We have not defined PWA characteristics yet. We have to make some time to add characteristics, even prepare a proper manifest.

Reply:
Definitely yes, we will check that feature in week 3.